packages/shared.net/src/createClientContext.ts

```diff
@@ -14,28 +14,25 @@
  * limitations under the License.
  */
 
+import type { SharedClientContext } from "@osdk/shared.client";
 import { createSharedClientContext } from "@osdk/shared.client.impl";
 import { PalantirApiError } from "@osdk/shared.net.errors";
 import {
   createFetchHeaderMutator,
   createFetchOrThrow,
   createRetryingFetch,
 } from "@osdk/shared.net.fetch";
-import type { ClientContext } from "./ClientContext.js";
 
 /**
  * The goal of the thin client is to provide a way to tree shake as much as possible.
  */
 
-export function createClientContext<
```

Comment:
Ah cool, this addresses my question from the previous PR, looks like we are removing it